sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/autorestTest.java

```diff
@@ -0,0 +1,364 @@
+package com.azure.data.tables;
```

Comment:
Names of test files should match class it is testing. `autorestTest` should be `AzureTableImplTest`